web-wallet/src/lib/components/Allocate/Allocate.svelte

```diff
@@ -403,7 +403,20 @@
     color: var(--error-color);
   }
 
-  hr.glyph:after {
+  .glyph {
+    margin: 1em 0;
+    height: 1px;
+  }
+
+  .glyph:after {
     content: "↑↓";
+    display: inline-block;
+    position: relative;
+    top: -1.2em;
+    color: var(--divider-border-color);
+    border: 1px solid var(--divider-border-color);
+    border-radius: 2em;
+    padding: 0.5em 1.25em;
+    background-color: var(--light-grey);
```

Comment:
Should have used `--divider-background-color` here as named colours shouldn't leak out of the language.

Reply:
Yeah, of course. It was just for debugging haha will fix it